src/sql/workbench/parts/charts/browser/chartView.ts

```diff
@@ -127,6 +143,13 @@ export class ChartView extends Disposable implements IPanelView {
 
 	}
 
+	// method that is used to generate array of user-friendly ones for display.
```

Comment:
Make sure you're using actual JSDoc comments - https://jsdoc.app/about-getting-started.html

If you type /** and hit enter above a method header it'll auto-create the comment for you (with params and whatnot already filled in)